population/README.md

```diff
@@ -11,25 +11,71 @@ End size: 200
 
 ## Populations
 
-Say we have a popluation of `n` llamas. Each year, `n/3` new llamas are born.
+Say we have a popluation of `n` llamas. Each year, `n/3` new llamas are born, and `n/4` llamas pass away.
 
- TODO
+For example, if we were to start with `n = 1200` llamas, then in that year, `1200/3` new llamas would be born and `1200/4` llamas would pass away. At the end of that year, we would have `1200 + 1200/3 - 1200/4 = 1200 + 400 - 300 = 1300` llamas. 
+
+As another example, if we were to start with `n = 1000` llamas, at the end of the year , we would have `1000 + 1000/3 - 1000/4 = 1083.33` llamas. We can't have a decimal portion of a llama though, so we'll truncate the decimal to get `1083`. The next year will then begin with `1083` llamas.
```

Comment:
@amazhangphyl, It's probably useful to clarify here that instead of 1000 + 1000/3 + 1000/4 = 1083.333 which truncates to 1583, it's really more that 1000 + 1000/3 - 1000/4 = 1000 + 333.333 - 250 truncates to 1000 + 333 - 250 = 1083.

That'll be relevant for something like `n = 1207` where 1207 + 1207/3 - 1207/4 = 1307.58 truncating to 1307 is incorrect; our solution would actually want 1207 + 1207/3 - 1207/4 = 1207 + 402.33 - 301.75 truncates to 1207 + 402 - 301 = 1308.